.semaphore/cleanup.yml

```diff
@@ -27,3 +27,18 @@ blocks:
       - ./.semaphore/clean-up-vms ${VM_PREFIX}
     secrets:
     - name: google-service-account-for-gce
+
+- name: Clean up windows resources
```

Comment:
```suggestion
- name: Clean up windows cni resources
```

Reply:
That is the job name for both cni and felix, the jobs are names accordingly with cni and felix